pkg/odo/cli/registry/update.go

```diff
@@ -0,0 +1,86 @@
+package registry
+
+import (
+	// Build-in packages
+	"fmt"
+
+	// Third-party packages
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	ktemplates "k8s.io/kubernetes/pkg/kubectl/util/templates"
+
+	// odo packages
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/odo/genericclioptions"
+	"github.com/openshift/odo/pkg/preference"
+)
+
+const updateCommandName = "update"
+
+// "odo registry update" command description and examples
+var (
+	updateLongDesc = ktemplates.LongDesc(`Update devfile registry`)
+
+	updateExample = ktemplates.Examples(`# Update devfile registry
+	%[1]s cheregistry https://che-devfile-registry-update.openshift.io/
+	`)
+)
+
+// UpdateOptions encapsulates the options for the "odo registry update" command
+type UpdateOptions struct {
+	operation    string
+	registryName string
+	registryURL  string
+}
+
+// NewUpdateOptions creates a new UpdateOptions instance
+func NewUpdateOptions() *UpdateOptions {
+	return &UpdateOptions{}
+}
+
+// Complete completes UpdateOptions after they've been created
+func (o *UpdateOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
+	o.operation = "update"
+	o.registryName = args[0]
+	o.registryURL = args[1]
+	return
+}
+
+// Validate validates the UpdateOptions based on completed values
+func (o *UpdateOptions) Validate() (err error) {
+	return
+}
+
+// Run contains the logic for "odo registry update" command
+func (o *UpdateOptions) Run() (err error) {
+
+	cfg, err := preference.New()
+	if err != nil {
+		return errors.Wrapf(err, "Unable to update registry")
+	}
+
+	err = cfg.RegistryHandler(o.operation, o.registryName, o.registryURL)
```

Comment:
maybe ask for the user's confirmation like we do for other command commands
https://github.com/openshift/odo/blob/master/pkg/odo/cli/url/delete.go#L111

Reply:
Done